../../../bin/pysondb,sha256=Bl_srwD-KVP1IqTDoRGzRPezHjMhBWiFaSrPRfWZ6TY,279
pysondb-1.6.7.dist-info/INSTALLER,sha256=zuuue4knoyJ-UwPPXg8fezS7VCrXJQrAP7zeNuwvFQg,4
pysondb-1.6.7.dist-info/LICENSE,sha256=UlvGFmhX3owSoCRpIUbh61KPl1tb1WOkFd2t75FHHnI,1067
pysondb-1.6.7.dist-info/METADATA,sha256=QAybc-B-NifyQ0oKf5b13aqY4Y_5Oktoi5c0D3ykS8U,5579
pysondb-1.6.7.dist-info/RECORD,,
pysondb-1.6.7.dist-info/REQUESTED,sha256=47DEQpj8HBSa-_TImW-5JCeuQeRkm5NMpJWZG3hSuFU,0
pysondb-1.6.7.dist-info/WHEEL,sha256=G16H4A3IeoQmnOrYV4ueZGKSjhipXx8zc8nu9FGlvMA,92
pysondb-1.6.7.dist-info/entry_points.txt,sha256=GflfXq4eGq0FCqXQ7ycDEAFe_u23O0qYRfYY-fZUoY8,46
pysondb-1.6.7.dist-info/top_level.txt,sha256=96iW8tJ80LJNH6cCPaKfkLTq4DgBEhhbKlWYc8qidS8,14
pysondb/__init__.py,sha256=lmHZVGMrkdNUQEYdQfODWW9AtZ5fTZCZaNLi7SlznHc,21
pysondb/__pycache__/__init__.cpython-312.pyc,,
pysondb/__pycache__/cli.cpython-312.pyc,,
pysondb/__pycache__/db.cpython-312.pyc,,
pysondb/cli.py,sha256=CobSJQ2d34NOW-4vTYKdJWuzfee4NuOqEKNnebj3rcU,7718
pysondb/db.py,sha256=pl0kPhbHtLCUw3YUDjGmucib3qn2xgMz39FInz86Umg,11809
pysondb/errors/__init__.py,sha256=oPfTBlIVNqOml1SW7kt71JK_lNwKVWQXXXfVn98iNeA,70
pysondb/errors/__pycache__/__init__.cpython-312.pyc,,
pysondb/errors/__pycache__/db_errors.cpython-312.pyc,,
pysondb/errors/db_errors.py,sha256=EwxoaQsIzslx_nlV4yp6QBa3YlXgXgFIHXXFSB5eCRI,852
tests/__init__.py,sha256=47DEQpj8HBSa-_TImW-5JCeuQeRkm5NMpJWZG3hSuFU,0
tests/__pycache__/__init__.cpython-312.pyc,,
